app/Http/Controllers/CompetitionsController.php

```diff
@@ -39,7 +48,9 @@ public function show(Competition $competition)
     {
         $contest = Contest::find($competition->contest_id);
 
-        return view('competitions.show', compact('competition', 'contest'));
+        $users = $this->repository->getAll($competition->users->pluck('id')->toArray());
```

Comment:
:scream: he used to be angry at me for that. maybe it's different when it's coming from the cache and I know nothing.